roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -48,8 +48,9 @@
       register: disabled_repos
       changed_when: false
 
-    - name: Enable powertools
-      command: dnf config-manager --set-enabled powertools
+    - name: Enable PowerTools
+      block:
+        - name: Enable PowerTools
+          command: dnf config-manager --set-enabled powertools
       when:
-        - ansible_distribution == 'CentOS'
```

Comment:
You're dropping this, to make it run on Rocky and Alma too, right?

I think then we should guard on `ansible_distribution != 'RedHat'`, as there is no powertools there.

Reply:
Ah, right, I didn't knew that :) added!